arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/precompiles/ModexpMetadata.java

```diff
@@ -121,11 +121,11 @@ public EWord rawLeadingWord() {
         nExponentBytesInCallData > 0
             && nExponentBytesInCallData < WORD_SIZE);
     StringBuilder callDataHexString = new StringBuilder(callData.slice(exponentOffsetInCallData, nExponentBytesInCallData).toHexString());
-    for (int i = 0; i < nExponentBytesInCallData; i++) {
+    for (int i = nExponentBytesInCallData; i < WORD_SIZE; i++) {
```

Comment:
@FlorianHuc @letypequividelespoubelles @powerslider what is the canonical way of doing this right padding ?

Reply:
Oh I may be thinking of

```java
 static Bytes32 rightPad(Bytes value)
```

Reply:
Should be fixed with https://github.com/Consensys/linea-tracer/pull/748/commits/1b5489df7b353775541c90ca2faeb2ec6df2bc9c